vendor/bat-native-ads/src/bat/ads/internal/transfer/transfer.cc

```diff
@@ -91,14 +91,14 @@ void Transfer::OnTransferAd(const int32_t tab_id,
     return;
   }
 
-  const absl::optional<TabInfo> tab =
-      TabManager::GetInstance()->GetTabForId(tab_id);
-  if (!tab) {
+  absl::optional<TabInfo> tab = TabManager::GetInstance()->GetTabForId(tab_id);
+  if (!tab || tab->redirect_chain.empty()) {
     FailedToTransferAd(ad);
     return;
   }
 
-  if (!DomainOrHostExists(redirect_chain, tab->url)) {
+  DCHECK(!redirect_chain.empty());
```

Comment:
this should be DCHECK(!tab->redirect_chain.empty());

Reply:
Fixed